### PR TITLE
Fix system delegation smart contract logs

### PR DESCRIPTION
**Pull request by miiu96**
- Fixed the delegation system smart contract logs

### CODE REVIEW

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

File: vm/systemSmartContracts/delegation.go [hunk 1]
```diff
@@ -1736,7 +1736,7 @@ func (d *delegation) unDelegate(args *vmcommon.ContractCallInput) vmcommon.Retur
 		return vmcommon.UserError
 	}
 
-	d.createAndAddLogEntry(args, valueToUnDelegate.Bytes(), remainedFund.Bytes(), globalFund.TotalActive.Bytes())
+	d.createAndAddLogEntry(args, valueToUnDelegate.Bytes(), remainedFund.Bytes(), big.NewInt(0).Bytes(), globalFund.TotalActive.Bytes())
```
Comment (iulianpascalau) [inline vm/systemSmartContracts/delegation.go:1739]: `big.NewInt(0).Bytes()` will return an empty byte slice. Is this correct?

Comment (miiu96) [reply vm/systemSmartContracts/delegation.go:1739]: Yes.

**Review by iulianpascalau — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.34-rc7
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/covalent-indexer-go v1.0.6
-	github.com/ElrondNetwork/elastic-indexer-go v1.1.38
+	github.com/ElrondNetwork/elastic-indexer-go v1.1.39-0.20220405170207-f2d05c0e40a6
```
Comment (iulianpascalau) [inline go.mod:11]: proper release

Comment (miiu96) [reply go.mod]: done.

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**